zrml/prediction-markets/src/migrations.rs

```diff
@@ -187,13 +198,13 @@ impl<T: Config + zrml_market_commons::Config> OnRuntimeUpgrade
             assert_eq!(new_market.report, old_market.report);
             assert_eq!(new_market.resolved_outcome, old_market.resolved_outcome);
             assert_eq!(new_market.dispute_mechanism, old_market.dispute_mechanism);
-            assert_eq!(new_market.bonds.oracle.unwrap().value, T::OracleBond::get());
```

Comment:
The `assert_eq!` for the `base_asset` can be replaced by a normal `assert_eq!(new_market.base_asset, old_market.base_asset)` at this point.